src/components/Selector.jsx

```diff
@@ -145,7 +144,14 @@ export default function Selector({templateInfo, animationManager, blinkManager,
   },[selectedOptions])
   // user selects an option
   const selectTraitOption = (option) => {
-    
+    if (option.key === 'class_1' || option.key === 'class_0') {
+      effectManager.setTransitionEffect('fade_out_avatar');
```

Comment:
I think we can set the effect below so it applies to every class in case we are more in future